aequilibrae/distribution/ipf.py

```diff
@@ -237,6 +247,39 @@ def fit(self):
             self.report.append("")
             self.report.append("Running time: " + str("{:4,.3f}".format(perf_counter() - t)) + "s")
 
+    def save_to_project(self, name: str, file_name: str) -> MatrixRecord:
+        """Saves the matrix output to the project file
+
+        Args:
```

Comment:
out of curiosity and not sure if would cause more headache than benefit. But I've seen in the skim saving similar opeartions of checking stuff, checking existence and so on. Although straighforward it might be the case to handle this type of operations in a different class (which could check extension, existence and return the record or part of it)

Reply:
True

Reply:
How does it look like now?

Reply:
I liked.